docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -476,7 +477,7 @@ To receive notifications whenever the device connection is established or lost:
    remoteDevice.removeConnectStateChangeListener(watchId);
    ```
 
-## Retrieving Bluetooth GATT Services
+### Retrieving Bluetooth GATT Services
```

Comment:
In 6.0, the name of service's UUID field has changed to serviceUuid - we need to change names used here.

Also, we need to mention the new characteristics' and descriptors' uuid field.